lambda/entry.go

```diff
@@ -18,31 +18,73 @@ import (
 //   - handler may take between 0 and two arguments.
 //   - if there are two arguments, the first argument must satisfy the "context.Context" interface.
 //   - handler may return between 0 and two arguments.
-//   - if there are two return values, the second argument must be an error.
+//   - if there are two return values, the second return value must be an error.
 //   - if there is one return value it must be an error.
 //
 // Valid function signatures:
 //
 //	func ()
+//	func (TIn)
 //	func () error
 //	func (TIn) error
 //	func () (TOut, error)
 //	func (TIn) (TOut, error)
-//	func (context.Context) error
+//	func (context.Context, TIn)
 //	func (context.Context, TIn) error
-//	func (context.Context) (TOut, error)
 //	func (context.Context, TIn) (TOut, error)
 //
 // Where "TIn" and "TOut" are types compatible with the "encoding/json" standard library.
 // See https://golang.org/pkg/encoding/json/#Unmarshal for how deserialization behaves
-func Start(handler interface{}) {
+func Start(handler any) {
```

Comment:
I don't want to drop support for all the older versions of go just yet, so we need to leave these and `interface{}` for now

Reply:
Ah, it seems I misunderstood how the go version in the go.mod file works. I thought it constrained the module to at least that version of go